examples/static-export/next-sitemap.config.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/** @type {import('next-sitemap').IConfig} */
+const config = {
+  siteUrl: process.env.SITE_URL || 'https://example.com',
+  generateRobotsTxt: true,
+  output: 'static', // Set static output here
```

Comment:
Shouldn't this be "export"?

Reply:
@ovflowd Thanks for pointing it out, fixed in example config

Reply:
You welcome :D